src/services/db-manager.ts

```diff
@@ -65,9 +65,12 @@ export function grantPostgresRoleQuery(user: string) {
   `;
 }
 
-export function dropPostgresRoleQuery(user: string) {
+export function dropPostgresRoleQuery(user: string, ephemeral = false) {
+  // Ephemeral users need to drop their owned properties first. I do not understand why only them
+  // and not the originally-created users for the database. Nor do I understand why dropping for
+  // the originally created users causes issues.
```

Comment:
what's the error message from the database when running `DROP OWNED BY ${user}`?

Reply:
This one: https://sentry.io/organizations/iasql/issues/3260895664/?project=6327420&referrer=webhooks_plugin

It's very strange.